In version 3.3.0 upstream started making use of features introduced
in macOS 10.11, which caused failures when building the MacOSX
backend on earlier macOS versions..

see: https://github.com/matplotlib/matplotlib/pull/17956

--- src/_macosx.m.orig	2020-07-18 17:50:09.000000000 -0400
+++ src/_macosx.m	2020-07-18 19:44:06.000000000 -0400
@@ -1207,8 +1207,12 @@
     rect.size.height = 0;
     rect.origin.x += height;
     NSTextView* messagebox = [[NSTextView alloc] initWithFrame: rect];
-    messagebox.textContainer.maximumNumberOfLines = 2;
-    messagebox.textContainer.lineBreakMode = NSLineBreakByTruncatingTail;
+
+    #if MAC_OS_X_VERSION_MAX_ALLOWED >= 101100 && MAC_OS_X_VERSION_MIN_REQUIRED >= 101100
+        messagebox.textContainer.maximumNumberOfLines = 2;
+        messagebox.textContainer.lineBreakMode = NSLineBreakByTruncatingTail;
+    #endif
+
     [messagebox setFont: font];
     [messagebox setDrawsBackground: NO];
     [messagebox setSelectable: NO];
